src/containerapp/azext_containerapp/java_component_decorator.py

```diff
@@ -28,15 +32,32 @@ def get_argument_configuration(self):
 
     def get_argument_environment_name(self):
         return self.get_param("environment_name")
+    
+    def get_environment_id(self, cmd: AzCliCommand):
```

Comment:
It seems that it is a common logic, is there any existing method that we can reuse?

Reply:
I didn't find one. I will take another look.